hack/tools/logcheck/testdata/src/doNotAllowUnstructuredLogs/doNotAllowUnstructuredLogs.go

```diff
@@ -41,6 +41,10 @@ func doNotAllowUnstructuredLogs() {
 	klog.ErrorS(nil, "Starting container in a pod", testKey, "containerID") // want `Key positional arguments are expected to be inlined constant strings. `
 	klog.InfoS("test: %s", "testname")                                      // want `structured logging function "InfoS" should not use format specifier "%s"`
 	klog.ErrorS(nil, "test no.: %d", 1)                                     // want `structured logging function "ErrorS" should not use format specifier "%d"`
+	klog.Exit("test log")
+	klog.ExitDepth(1, "test log")
+	klog.Exitln("test log")
+	klog.Exitf("test log")
```

Comment:
Those lines should not be allowed, meaning they should have comment 
```
// want `unstructured logging function "Infof" should not be used`
```
Similar to lines below